main.go

```diff
@@ -52,14 +113,118 @@ func pong(c *gin.Context) {
 	})
 }
 
+func generateCode() (int, error) {
+	max := big.NewInt(10000)
+
+	r, err := rand.Int(rand.Reader, max)
+	if err != nil {
+		return 0, err
+	}
+
+	return int(r.Int64()), nil
+}
+
+func generateRandBytes() ([]byte, error) {
+	c := 10
+	b := make([]byte, c)
+	_, err := rand.Read(b)
```

Comment:
Where did you find this method for generating random bytes? Seems incredibly dubious, the bytes here are probably undefined not random. This is a security concern

Reply:
That's from [here](https://golang.org/pkg/crypto/rand/#example_Read), but I don't need that function anyway so I should get rid of it

Reply:
Oh yeah that actually looks fine I just misread the code